contrib/windows/msys_build.sh

```diff
@@ -206,3 +206,4 @@ cat Make.user
 make -j3 VERBOSE=1
 make build-stats
 #make debug
+make prefix=/c/projects/julia/usr VERBOSE=1 -C examples
```

Comment:
Ah okay it looks like `prefix` gets set relative to `examples` when `examples/Makefile` includes `Make.inc`. If we were to fix that, then `make VERBOSE=1 install && make VERBOSE=1 -C examples` in `msys_build.sh` would hopefully work.